src/containers/Downloads/DownloadStart/EmailForm.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { reduxForm, Field, formValueSelector } from 'redux-form';
+import { connect } from 'react-redux';
+import { CheckboxField } from '../../../components/Checkbox';
+import Button from '../../../components/Button';
+import { Link } from 'react-router-dom';
+
+/**
+ * This form can be used to edit the email that's associated with a dataset
+ */
+let EmailForm = ({ handleSubmit, isSubmitDisabled, agreedToTerms }) => {
```

Comment:
Since the arrow function's body returns the object directly, I think the code can be slightly simplified into:
```
let EmailForm = ({ handleSubmit, isSubmitDisabled, agreedToTerms }) => (
  <form onSubmit={handleSubmit}>
  ...
  </form>
);
```